self-core-impl/src/test/java/com/selfxdsd/core/contributors/ProjectContributorsTestCase.java

```diff
@@ -275,28 +276,28 @@ public void electsNewContributorForAssignedTask() {
     @Test
     public void electsNewContributorForUnassignedTask() {
         final Contributors contributors = new ProjectContributors(
-            "john/test", Provider.Names.GITHUB,
-            List.of(
-                this.mockContributor("mihai", "DEV", "REV", "QA"),
-                this.mockContributor("vlad", "DEV"),
-                this.mockContributor("mary", "REV", "QA"),
-                this.mockContributor("george", "DEV", "ARCH"),
-                this.mockContributor("karen", "DEV")
-            ),
-            Mockito.mock(Storage.class)
+                "john/test", Provider.Names.GITHUB,
+                List.of(
+                        this.mockContributor("mihai", "DEV", "REV", "QA"),
+                        this.mockContributor("vlad", "DEV"),
+                        this.mockContributor("mary", "REV", "QA"),
+                        this.mockContributor("george", "DEV", "ARCH"),
+                        this.mockContributor("karen", "DEV")
+                )::stream,
+                Mockito.mock(Storage.class)
         );
         final Task task = Mockito.mock(Task.class);
         Mockito.when(task.assignee()).thenReturn(null);
         Mockito.when(task.role()).thenReturn("DEV");
         final Contributor elected = contributors.elect(task);
 
         MatcherAssert.assertThat(
-            elected.username(),
-            Matchers.not(Matchers.equalTo("mary"))
+                elected.username(),
+                Matchers.not(Matchers.equalTo("mary"))
         );
         MatcherAssert.assertThat(
-            elected.username(),
-            Matchers.isOneOf("mihai", "vlad", "george", "karen")
+                elected.username(),
+                Matchers.isOneOf("mihai", "vlad", "george", "karen")
         );
```

Comment:
@criske indentation issues here, can you please indent them by 4 spaces instead of 8? :D